test/minter-suite-minters/MinterHolder/MinterHolderV1.test.ts

```diff
@@ -185,6 +185,133 @@ describe("MinterHolderV1", async function () {
+      await this.genArt721Core
+        .connect(this.accounts.deployer)
+        .updateArtblocksPrimarySalesAddress(this.deadReceiver.address);
+      // expect revert when trying to purchase
+      await expectRevert(
+        this.minter
+          .connect(this.accounts.artist)
+          .purchase_nnf(
+            this.projectZero,
+            this.genArt721Core.address,
+            this.projectZeroTokenZero.toNumber(),
+            {
+              value: this.pricePerTokenInWei,
+            }
+          ),
+        "Art Blocks payment failed"
+      );
+    });
+
+    it("requires successful payment to artist", async function () {
+      // update artist address to a contract that reverts on receive
+      await this.genArt721Core
+        .connect(this.accounts.deployer)
+        .updateProjectArtistAddress(
+          this.projectZero,
+          this.deadReceiver.address
+        );
+      // expect revert when trying to purchase
+      await expectRevert(
+        this.minter
+          .connect(this.accounts.artist)
+          .purchase_nnf(
+            this.projectZero,
+            this.genArt721Core.address,
+            this.projectZeroTokenZero.toNumber(),
+            {
+              value: this.pricePerTokenInWei,
+            }
+          ),
+        "Artist payment failed"
+      );
+    });
+
+    it("requires successful payment to artist additional payee", async function () {
+      // update artist additional payee to a contract that reverts on receive
+      const proposedAddressesAndSplits = [
+        this.projectZero,
+        this.accounts.artist.address,
+        this.deadReceiver.address,
+        50,
+        this.accounts.additional2.address,
+        50,
+      ];
+      await this.genArt721Core
+        .connect(this.accounts.artist)
+        .proposeArtistPaymentAddressesAndSplits(...proposedAddressesAndSplits);
+      await this.genArt721Core
+        .connect(this.accounts.deployer)
+        .adminAcceptArtistAddressesAndSplits(...proposedAddressesAndSplits);
+      // expect revert when trying to purchase
+      await expectRevert(
+        this.minter
+          .connect(this.accounts.artist)
+          .purchase_nnf(
+            this.projectZero,
+            this.genArt721Core.address,
+            this.projectZeroTokenZero.toNumber(),
+            {
+              value: this.pricePerTokenInWei,
+            }
+          ),
+        "Additional Payee payment failed"
+      );
+    });
+
+    it("handles zero platform and artist payment values", async function () {
+      // update platform to zero percent
+      await this.genArt721Core
+        .connect(this.accounts.deployer)
+        .updateArtblocksPrimarySalesPercentage(0);
+      // update artist primary split to zero
+      const proposedAddressesAndSplits = [
+        this.projectZero,
+        this.accounts.artist.address,
+        this.accounts.additional.address,
+        100,
+        this.accounts.additional2.address,
+        50,
```

Comment:
Nit: these two constant values could be helpful to add smol dev comments for.

Reply:
updated 🙏